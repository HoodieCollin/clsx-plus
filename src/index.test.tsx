import { expect, it } from '@jest/globals';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import tw from './index';

it('renders correctly', () => {
  const { asFragment } = render(<div className={tw`w-px h-2`()} />);

  expect(asFragment()).toMatchSnapshot();
});

it('renders inline styles', () => {
  const { asFragment } = render(
    <div
      {...tw`
        w-px h-2
        ${tw.css`color: red`}
      `}
    />
  );

  expect(asFragment()).toMatchSnapshot();
});
