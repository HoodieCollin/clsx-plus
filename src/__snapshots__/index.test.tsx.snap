// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="w-px h-2"
  />
</DocumentFragment>
`;

exports[`renders inline styles 1`] = `
<DocumentFragment>
  <div
    class="w-px h-2"
    style="color: red;"
  />
</DocumentFragment>
`;
